pinot-core/src/main/java/org/apache/pinot/core/query/request/context/utils/BrokerRequestToQueryContextConverter.java

```diff
@@ -95,6 +96,8 @@ private static QueryContext convertSQL(BrokerRequest brokerRequest) {
       for (Expression thriftExpression : groupByList) {
         groupByExpressions.add(RequestContextUtils.getExpression(thriftExpression));
       }
+      pinotQuery.putToQueryOptions(CommonConstants.Broker.Request.QueryOptionKey.GROUP_BY_MODE,
```

Comment:
This should not be required because we have this check on the broker side to ensure the SQL query always have this option set